tests/tcp.rs

```diff
@@ -1229,4 +1248,13 @@ fn tcp_no_events_after_deregister() {
     let mut buf = [0; 10];
     assert_eq!(stream.read(&mut buf).unwrap(), 4);
     assert_eq!(&buf[0..4], &[1, 2, 3, 4]);
+
+    stream2.write_all(&[1, 2, 3, 4]).unwrap();
+
+    expect_no_events_with_timeout(&mut poll, &mut events, Some(Duration::from_millis(200)));
```

Comment:
Why does this need a special timeout value? I rather not have an additional function for this.

Reply:
I want to make sure there's enough delay between the previous write and this read, so the read will complete successfuly. That's why I had the timeout in the first place. I think having two versions of expect_no_events, one with timeout exposed will be usefull for future tests. It's weird that both expect_no_events and expect_events do poll with timeout inside, we might want to expose them both once the tests took of and we write many more tests. That's my thinking atm.

Reply:
On second thought I think you had it right the first time round. Lets change this back to `sleep(Duration::from_millis(200))`, you can import `std::thread::sleep` at the top. Sorry for this distraction.

Reply:
np, I've reverted the last change